src/main/java/org/isf/lab/manager/LabManager.java

```diff
@@ -266,6 +267,41 @@ public boolean newLaboratory2(Laboratory laboratory, List<LaboratoryRow> labRow)
 		}
 	}
 
+	/**
+	 * Inserts one Laboratory request (All Procedures)
+	 *
+	 * @param laboratory - the laboratory without result
+	 * @return <code>true</code> if the request has been inserted, <code>false</code> otherwise
+	 * @throws OHServiceException
+	 */
+	public boolean newExamRequest(Laboratory laboratory) throws OHServiceException {
+		setPatientConsistency(laboratory);
+		LaboratoryStatus laboratoryStatus = LaboratoryStatus.DRAFT;
+		laboratory.setStatus(laboratoryStatus.toString());
+		laboratory.setResult("");
+		return ioOperations.newLabFirstProcedure(laboratory);
+	}
+	
+	/**
+	 * update one Laboratory request {@link Laboratory} (All Procedures)
+	 *
+	 * @param laboratory - the laboratory without result
+	 * @return <code>true</code> if the request has been update, <code>false</code> otherwise
+	 * @throws OHServiceException
+	 */
+	public boolean updateExamRequest(int code, LaboratoryStatus status) throws OHServiceException {
+		Optional<Laboratory> laboratory = ioOperations.getLaboratory(code);
+		if (laboratory.isPresent()) {
+			Laboratory lab = laboratory.get();
+			lab.setStatus(status.toString());
+			return ioOperations.updateLabFirstProcedure(lab);
+		} else {
```

Comment:
No need for the `else` when the previous statement is a `return`

Reply:
Let's keep it this way for readability, thanks!

Reply:
FWIW, other methods have it without the `else`; just trying to get things consistent and without too much indentation 😏